airflow/contrib/sensors/redis_pub_sub_sensor.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, channels, redis_conn_id, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.channels = channels
         self.redis_conn_id = redis_conn_id
-        self.pubsub = RedisHook(redis_conn_id=self.redis_conn_id).get_conn().pubsub()
+        self.pubsub = RedisHook(redis_conn_id=self.redis_conn_id).create_pubsub()
```

Comment:
It would be nice to move this invocation to `poke` method.

Reply:
It made sense when we discussed it but playing around with Redis for a little bit it actually turns out, that maybe it should stay like that. The reason for this is when you create subscription it only reads messages published after subscription is created. So if message was published between two pokes, it would be lost. What do you think ?

Reply:
That's interesting and I can agree. Additionally this could be a problem when used with `reschedule` mode. WDYT @mik-laj ?

Reply:
Creating connections in Execute is a _hard_ no. This code will get called _every time Airflow parses your dag file_

Reply:
You can do this though:

```
def execute(self, **context):
    self.pubsub = RedisHook(redis_conn_id=self.redis_conn_id).create_pubsub()
    return super().execute(**context)
```

(untested though, but should be roughly right)

Reply:
You mean 'Creating connections in **\_\_init\_\_** is a _hard_ no', right ? thanks for this suggestion, I've moved it and aligned tests accordingly.